ERCS/erc-7196.md

```diff
@@ -0,0 +1,162 @@
+---
+eip: 7196
+title: simple token
+description: Designed for smart contract wallet
+author: Xiang (@wenzhenxiang), Ben77 (@ben2077), Mingshi S. (@newnewsms)
+discussions-to: https://ethereum-magicians.org/t/simple-token-designed-for-smart-contract-wallet-aa/14757
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-06-21
+---
```

Comment:
```suggestion
requires: 20
---
```

Enough of your proposal is based on ERC-20, so requiring it makes sense. Note that this doesn't mean implementers have to implement the `ERC20` interface.